README.md

```diff
@@ -18,4 +18,18 @@ TOKEN = "<INSERT YOUR TOKEN HERE>"
 
 5. Add your own bot to a discord server (you can make a dev server for testing your bots).
 6. Run `npm install` to install node packages.
-7. Run `node index.js` then check if the bot will go online in discord.
\ No newline at end of file
+7. Run `node index.js` then check if the bot will go online in discord.
+
+## UID Link
+The `?uid` command allows you to link a discord user to a given Genshin Impact UID. This would require you to have a linked firestore database and add your credentials to the env file.
+
+### Required .env Variables
+```
+FIREBASE_API_KEY
+FIREBASE_APP_ID
+FIREBASE_PROJECT_ID
+```
+
```

Comment:
Can you add links on how/where to generate the firebase variables? Assume that future developers who will work on this has zero knowledge on firebase.